distributed/client.py

```diff
@@ -499,8 +504,16 @@ def release(self):
             except TypeError:  # pragma: no cover
                 pass  # Shutting down, add_callback may be None
 
+    @staticmethod
+    def make_future(key, id):
+        # Can't use kwargs in pickle __reduce__ methods
+        return Future(key=key, _id=id)
+
     def __reduce__(self) -> str | tuple[Any, ...]:
-        return Future, (self.key,)
+        return Future.make_future, (self.key, self._id)
+
+    def __dask_tokenize__(self):
+        return (type(self).__name__, self.key, self._id)
```

Comment:
```suggestion
        return (normalize_token(type(self)), self.key, self._id)
```
nit - mostly in case a third party subclasses `distributed.Future` without changing the name